Jagati.cmake

```diff
@@ -2085,14 +2093,14 @@ public:\n\
 \n\
     /// @brief Get the error message in the std compatible way.\n\
     /// @return A pointer to a C-String, containing the same messages as @ref GetMessage().\n\
-    virtual const char* what() const noexcept\n\
+    virtual const char* what() const noexcept override\n\
```

Comment:
I'm assuming instead of using the default exception message, line 2096 overrides the standard exception message to return a custom error message.

Reply:
Exactly!

This override is analagous to Java's `@override`, but is part of the core language.